Framework/Core/include/Framework/AnalysisDataModel.h

```diff
@@ -1672,6 +1672,26 @@ DECLARE_SOA_COLUMN(SPDFiredFastOrL0, spdFiredFastOrL0, uint16_t);     //! Fired
 DECLARE_SOA_COLUMN(SPDFiredFastOrL1, spdFiredFastOrL1, uint16_t);     //! Fired FASTOR signals in the first layer of the SPD (online)
 DECLARE_SOA_COLUMN(V0TriggerChargeA, v0TriggerChargeA, uint16_t);     //! V0A trigger charge
 DECLARE_SOA_COLUMN(V0TriggerChargeC, v0TriggerChargeC, uint16_t);     //! V0C trigger charge
+namespace v0otf
```

Comment:
```suggestion
namespace run2v0otf
```

Reply:
@jgrosseo I have the v0otf namespace inside the run2 namespace that's why I didn't add the run2 there explicitly.
So should I move outside the run2 namespace, or should I just add the run2 anyways?